twarep/electrifly
# Down sampled flight data for performance optimization

## Checklist

## What I Did
- Edited the storage file to down sample data before storing it.
- Edited the ml data to make sure that the program runs and labels the correct flight activity
- Changed `int8` to `float8` for the data type of the sql database to match the averaged values from the down sampling.
- Got rid of un-needed files

## How I Did it?
- Used python algorithmic knowledge with Pandas and Numpy.

## How to Test?
- Manually go into the database and look at the different flights, You'll see that the time goes up every 0.02 minutes.
- Run app.py to make sure everything is working well.

### Relevant Links
Any tutorials or other resources used to help with this task. You can also cite the documentation.
